app/app/ReviewComment.php

```diff
@@ -18,6 +18,21 @@ public function __construct() {
 		$this->attributes = array('id' => Uuid::generate(4)->string);
 	}
 
+	public function getQuestionCategory()
+	{
+	$this->question_category = $this->questionCategory()->first();
```

Comment:
Indent this a bit more so it looks good in the github change comparison tool.  You might have some regular spaces and tabs mixed.   If you copy and paste the spaces from elsewhere, this can help.